src/webgl/shaders/lighting.glsl

```diff
@@ -67,6 +75,44 @@ LightResult _light(vec3 viewDirection, vec3 normal, vec3 lightVector) {
   return lr;
 }
 
+float map(float value, float min1, float max1, float min2, float max2) {
```

Comment:
nit: A clearer name for this if possible

Reply:
The `map(vale, min1, min2, max1, max2)` function is converting the range of value from [min1 to max1] to [min2 to max2].

So should we make it something like `convertRange()` or `mapRange()` maybe?

Reply:
This function mirrors https://p5js.org/reference/#/p5/map so maybe we could link to that reference page + add a comment description of what it does?